### ISSUE TITLE
When deploying, CSS isn't linked correctly.

### ISSUE THREAD

**Issue by n3a9** [nielsenramon/chalk#133]
When forking and deploying immediately, the website will look like this. (Can be found [here](https://n3a9.github.io/chalk/).)

<img width="1920" alt="screen shot 2018-01-04 at 12 08 19 pm" src="https://user-images.githubusercontent.com/7104017/34582204-070f923e-f148-11e7-9a44-a68cab4fad17.png">

This is because, in the production index.html, it links to

[https://n3a9.github.io/assets/light-dd81b21ab090e25ca3de9c7f8a04c3ed3092374c826eb42b9fb7cac72c5f41e7.css](https://n3a9.github.io/assets/light-dd81b21ab090e25ca3de9c7f8a04c3ed3092374c826eb42b9fb7cac72c5f41e7.css),

instead of

[https://n3a9.github.io/chalk/assets/light-dd81b21ab090e25ca3de9c7f8a04c3ed3092374c826eb42b9fb7cac72c5f41e7.css](https://n3a9.github.io/chalk/assets/light-dd81b21ab090e25ca3de9c7f8a04c3ed3092374c826eb42b9fb7cac72c5f41e7.css).

In short, it is missing the link to the repository at the end of the link.

**Comment by migueldemoura** [comment]
Could you post your `_config.yml` here? Specifically the `url` and `baseurl` keys.

**Comment by n3a9** [comment]
Ah, found the issue. Was using `url:` instead of `baseurl:` inside `_config.yml`.

I set `baseurl: /chalk` and it worked. Thank you!

**Comment by migueldemoura** [comment]
Glad it is fixed!
